packages/teleport/src/console/useOnAltKeyDown.ts

```diff
@@ -0,0 +1,59 @@
+/**
+ * Copyright 2020 Gravitational, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+import ConsoleContext from './consoleContext';
+
+// DIGITS defines event.key values on keyboard from 1 - 9.
+export const DIGITS = ['1', '2', '3', '4', '5', '6', '7', '8', '9'];
+
+/**
+ * useOnAltKeyDown switches current console tab to the tab that
+ * user indicated by numbers 1-9.
+ *
+ * @param ctx data that is shared between Console related components
+ */
+const useOnAltKeyDown = (ctx: ConsoleContext) => {
+  React.useEffect(() => {
+    /**
+     * handleKeyDown listens and handles keyboard events: alt/command + <1-9>.
+     * Checks if alt or cmd was pressed with a number and if a document exists
+     * on the indicated tab, it will go to that tab.
+     */
+    const handleKeyDown = event => {
+      // windows/ubuntu uses alt key
+      // mac uses command key (meta key)
+      if (!event.altKey && !event.metaKey) return;
```

Comment:
I wouldn't use meta key here. "Meta+Number" switches between browser tabs, it's similar to "Ctrl+Number" on Linux. Browsers already map Mac's option key to `altKey` (see https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/altKey), so I would just use `altKey` for both systems

Reply:
> "HotKey" is a pretty ambiguous term introduced by Windows. "ModifierKey" would also work.

This function is about quick navigation on some "hotKey" command. HotKey these days mean a combination of keys providing quick access to a particular function within a program. Calling this function (which handles quick navigation) as ModifiyKey or AltKey feels like a leaked technical implementation.

Reply:
We can also name it as `useKeyboardNav`

Reply:
removed meta key